lib/services/flowdock.ts

```diff
@@ -245,22 +299,34 @@ export class FlowdockService extends MessageService implements ServiceEmitter, S
 }
 
 /**
- * Return this class activated and typed as a listener
+ * Build this class, typed and activated as a listener
+ * @returns {ServiceListener}
  */
 export function createServiceListener(): ServiceListener {
     return new FlowdockService(true);
 }
 
 /**
- * Return this class typed as an emitter
+ * Build this class, typed as an emitter
+ * @returns {ServiceEmitter}
  */
 export function createServiceEmitter(): ServiceEmitter {
     return new FlowdockService(false);
 }
 
 /**
- * Return this class typed as a messenger
+ * Build this class, typed as a message service
+ * @returns {MessageService}
  */
 export function createMessageService(): MessageService {
     return new FlowdockService(false);
 }
+
+//noinspection JSUnusedGlobalSymbols
```

Comment:
Whilst I understand the motives of taking out the `.vscode` directory (and in the long run, I don't particularly disagree), it's self-contained and nothing in the source has been altered to ensure it works. I *definitely* disagree with adding IDE specific comments in code.